gateway-server/src/main/java/org/apache/knox/gateway/websockets/ProxyWebSocketAdapter.java

```diff
@@ -81,6 +81,10 @@ public void onWebSocketConnect(final Session frontEndSession) {
      * plumbing takes place
      */
     container = ContainerProvider.getWebSocketContainer();
```

Comment:
I might should be casting this to ClientContainer so I can actually set the profile completely. I checked the jetty code and set max buffer size also sets max size.